packit/distgit.py

```diff
@@ -139,6 +153,11 @@ def create_pull(
         """
         project = self.local_project.git_project
 
+        if not self.pagure_user_token:
+            raise RuntimeError("Please provide PAGURE_USER_TOKEN as an environment variable.")
+        if not self.pagure_fork_token:
+            raise RuntimeError("Please provide PAGURE_FORK_TOKEN as an environment variable.")
```

Comment:
We can use custom exceptions also here. (Feel free to create a new one for this.)

Reply:
Will do, I got lazy.